src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -643,5 +643,7 @@ static String getResource(String key) {
             {"R_pvkParseError", "Could not read Private Key from PVK, check the password provided."},
             {"R_pvkHeaderError", "Cannot parse the PVK, PVK file does not contain the correct header."},
             {"R_clientCertError", "Reading client certificate failed. Please verify the location of the certificate."},
-            {"R_unassignableError", "The class specified by the {0} property must be assignable to {1}."}};
+            {"R_unassignableError", "The class specified by the {0} property must be assignable to {1}."},
+            {"R_delayLoadingLobsPropertyDescription",
+            "Boolean flag which indicates whether the driver will load LOB datatypes into memeory."}};
```

Comment:
We usually put descriptions along with other descriptions, so to not mix errors and descriptions together. Can you move this description to where the other descriptions are?